en/docs/security-guidelines/secure-engineering-guidelines/dynamic-analysis-with-owasp-zap.md

```diff
@@ -16,199 +16,199 @@ This document provides details of all necessary steps for configuring the OWASP
 
-Go to **Tools** &rarr; **Options** and select Global Exclude URL option. By default there are some patterns already added to ZAP. You can select all of them for exclusion. Additionally, if there is any URL pattern you need to exclude, you can add the regex for the URL as a new entry. 
+Go to **Tools** &rarr; **Options** and select the Global Exclude URL option. By default, there are some patterns already added to ZAP. You can select all of them for exclusion. Additionally, if there is any URL pattern you need to exclude, you can add the regex for the URL as a new entry. 
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-18.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-18.png)
 
 
 ### Creating the Logged in User Session
-When running the spider to crawl the site, we have to let ZAP discover the URLs that are accessible only by logged in users as well. For that, we need to create a logged in user session in ZAP, so that same as a logged in user can browse the URLs in a web browser, ZAP will be able to crawl through those URLs. 
+When running the spider to crawl the site, we have to let ZAP discover the URLs that are accessible only by logged-in users as well. For that, we need to create a logged-in user session in ZAP, so that same as a logged-in user can browse the URLs in a web browser, ZAP will be able to crawl through those URLs. 
 
-Click on **Show All Tabs** in the toolbox so that it will display all the tabs. 
+Click  **Show All Tabs** in the toolbox so that it will display all the tabs. 
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-19.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-19.png)
 
 Go to the **Http Sessions** tab. If there are already created sessions listed, you can remove them. 
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-20.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-20.png)
 
-Now while ZAP proxy is tracing the traffic, go to the browser and login to the site you need to scan. (When ZAP performs the scan, it will attack the URLs with the associated privileges of the user you logged in). Once you login, the session ID should be listed in the **HTTP Sessions** tab. Right click on the session and **Set as Active**. 
+Now while the ZAP proxy is tracing the traffic, go to the browser and log in to the site you need to scan. (When ZAP performs the scan, it will attack the URLs with the associated privileges of the user you logged in to). Once you log in, the session ID should be listed in the **HTTP Sessions** tab. Right-click the session and **Set as Active**. 
```

Comment:
```suggestion
Now, while the ZAP proxy is tracing the traffic, go to the browser and log in to the site you need to scan. Note that when ZAP performs the scan, it will attack the URLs with the associated privileges of the user you logged in as. Once you log in, the session ID should be listed in the **HTTP Sessions** tab. Right-click the session and **Set as Active**. 
```